src/components/add-book/AddBook.jsx

```diff
@@ -1,14 +1,29 @@
-const AddBook = () => (
-  <div>
-    <form>
-      <h2>Add a Book</h2>
-      <p>Title:</p>
-      <input type="text" id="title" placeholder="Add Title" />
-      <p>Author:</p>
-      <input type="text" id="Author" placeholder="Add Author" />
-      <button type="submit">Add Book</button>
-    </form>
-  </div>
-);
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../redux/books/books';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const { target } = e;
+    const title = target.title.value;
+    const author = target.author.value;
+    dispatch(addBook({ title, author }));
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <h2>Add a Book</h2>
+        <p>Title:</p>
+        <input type="text" id="title" placeholder="Add Title" />
+        <p>Author:</p>
+        <input type="text" id="author" placeholder="Add Author" />
```

Comment:
**OPTIONAL**

- You can add the `required` attribute to prevent users from adding empty books.

Reply:
Nice catch 👍 Thanks